### PR TITLE
Makes cigs branded too

### PR DESCRIPTION
**Pull request by Yoshax**
Description of cig/cigbutt would reveal its brand for dastardly murdersolving revelations.

Baystation12/Baystation12#14261

### CODE REVIEW

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -186,11 +191,13 @@
 	name = "\improper DromedaryCo packet"
 	desc = "A packet of six imported DromedaryCo cancer sticks. A label on the packaging reads, \"Wouldn't a slow death make a change?\""
 	icon_state = "Dpacket"
+	brand = "\improper Dromedary Co. cigarette"
```
Comment (Anewbe) [inline code/game/objects/items/weapons/storage/fancy.dm]: These descriptions aren't consistent with themselves, some have cigarette at the end and the others don't. I at least would prefer that they all use one or the other.

Comment (Yoshax) [inline code/game/objects/items/weapons/storage/fancy.dm]: I feel they read fine as is, its flow better with the cigarette on the two its on.

Comment (redstryker) [inline code/game/objects/items/weapons/storage/fancy.dm]: Yeah, they read pretty well, IMO.

Comment (Anewbe) [inline code/game/objects/items/weapons/storage/fancy.dm]: *shrug

**Pull request merged**